proposals/flexible-vectors/FlexibleVectors.md

```diff
@@ -0,0 +1,327 @@
+Flexible vectors overview
+=========================
+
+The goal of this proposal is to provide flexible vector instructions for
+WebAssembly as a way to bridge the gap between existing SIMD instruction sets
```

Comment:
This proposal could also enables better use of vector processing capabilities of the underlying platforms narrowing the WASM performance gap vs. respective native SIMD implementations.

Reply:
Good point.

Reply:
I've added that to updated doc.